URA-695/run_sentieon-tnbam.py

```
@@ -0,0 +1,102 @@
+        'genome_fastagz': dx.dxlink('file-Gb757784XGyY3FPvkPQ74K9z'),
+        'run_bqsr': True,
+        'somatic_algo': 'TNhaplotyper2'
+        }
+    
+    # Set the job for production variant caller
+    job_input['mappings_bam']= dx.dxlink(sample['sentieon-bwa_v3.2.0'])
+    prod_job = dx.DXApp(name="sentieon-tnbam", alias="3.2.0").run(
+        job_input,
+        project="project-Gjz0P404fV6bq4J30qBfPBFf",
+        folder="/sentieon-tnbam_v3.2.0",
+        name=f"sentieon-tnbam_v3.2.0_{sample['Name']}"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)